README.md

```diff
@@ -39,22 +36,18 @@ Work needed includes:
 | Platform neutrality   |               |                                                  |
 
 ### Logistics
-
 - Monthly Meetings
 - Biannual F2F
 
 ### Members
 
+Current Members#
+
 <!-- ncu-team-sync.team(nodejs/diagnostics) -->
 
 - [@AndreasMadsen](https://github.com/AndreasMadsen) - Andreas Madsen
-- [@bnoordhuis](https://github.com/bnoordhuis) - Ben Noordhuis
-- [@BridgeAR](https://github.com/BridgeAR) - Ruben Bridgewater
```

Comment:
Hm, I was added recently. Was it a mistake that I was added to this list?

Reply:
Yes, my mistake, you were added after the review started and I messed up. Will fix.